.travis.yml

```diff
@@ -8,9 +8,14 @@ cache:
   directories:
     - "${HOME}/.m2/"
 
+before_install:
+- openssl aes-256-cbc -K $encrypted_58a7d4f10478_key -iv $encrypted_58a7d4f10478_iv
```

Comment:
Personally, I prefer to use long opts in scripts, because when you look at it in a few weeks, you often don't remember what a particular option does (e.g. `-d` at the end).

Reply:
This is what was autogenerated by the command `travis encrypt-file retest-gmbh-gpg.asc --add`...